pkgs/applications/editors/quilter/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, fetchpatch, vala, pkgconfig, meson, ninja, granite
+, gtk3, desktop-file-utils, gnome3, gtksourceview, webkitgtk, gtkspell3, discount
+, gobjectIntrospection, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "quilter";
+  version = "1.6.3";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "lainsce";
+    repo = pname;
+    rev = version;
+    sha256 = "1wa0i6dgg6fgb7q9z33v9qmn1a1dn3ik58v1f3a49dvd5xyf8q6q";
+  };
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gobjectIntrospection
+    meson
+    ninja
+    pkgconfig
+    vala
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    discount
+    granite
+    gtk3
+    gtksourceview
+    gtkspell3
+    webkitgtk
+    gnome3.libgee
+  ];
+
+  LIBRARY_PATH = stdenv.lib.makeLibraryPath [ discount ];
```

Comment:
This is a bug in the project, `find_library` expects the library name without the `lib` prefix [here](https://github.com/lainsce/quilter/blob/5ecc3e400e6870ccba5c629ce91c9c9f7851de53/meson.build#L42).

Reply:
Please elaborate if you can?

Reply:
For an example, see https://gitlab.gnome.org/GNOME/tracker-miners/commit/02497b42bab5ff4f2f5a9c956cec0057a2adc7e5

Reply:
https://github.com/lainsce/quilter/pull/170